### PR TITLE
Run mariadb kuttl tests in a specific namespace

### PR DESCRIPTION
**Pull request by kajinamit**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1228,9 +1229,12 @@ nova_deploy_cleanup: namespace ## cleans up the service instance, Does not affec
 
 .PHONY: mariadb_kuttl_run
 mariadb_kuttl_run: ## runs kuttl tests for the mariadb operator, assumes that everything needed for running the test was deployed beforehand.
-	kubectl-kuttl test --config ${MARIADB_KUTTL_CONF} ${MARIADB_KUTTL_DIR}
+	MARIADB_KUTTL_DIR=${MARIADB_KUTTL_DIR} kubectl-kuttl test --config ${MARIADB_KUTTL_CONF} ${MARIADB_KUTTL_DIR} --namespace ${NAMESPACE}
```
Comment (cescgina) [inline Makefile:1232]: A quick grep does not show any use for `MARIADB_KUTTL_DIR` in the tests, I don't think we need it here

Comment (kajinamit) [reply Makefile:1232]: You are correct and this is not needed.

I was wondering adding this consistently would be helpful for future test implementation which would require path to some test files, but I can drop this now.

**Comment by kajinamit** [comment]
We might want to hold this during the dev preview release process, because we are close to tag mariadb-operator.

File: Makefile [hunk 2]
```diff
@@ -101,18 +101,24 @@ KEYSTONE_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/keystone-operator/tests/kuttl/t
...
+endif
+MARIADB_CR              ?= ${OPERATOR_BASE_DIR}/mariadb-operator/${MARIADB}
+MARIADB_DEPL_IMG        ?= unused
+MARIADB_KUTTL_CONF      ?= ${OPERATOR_BASE_DIR}/mariadb-operator/kuttl-test.yaml
+MARIADB_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/mariadb-operator/tests/kuttl/tests
+MARIADB_KUTTL_NAMESPACE ?= mariadb-kuttl-tests
+ifeq ($(DBSERVICE), galera)
+MARIADB_ASSERT          ?= ${MARIADB_KUTTL_DIR}/galera_deploy/01-assert.yaml
+else
+MARIADB_ASSERT          ?= ${MARIADB_KUTTL_DIR}/mariadb_deploy/01-assert.yaml
```
Comment (abays) [inline Makefile:120]: Is `MARIADB_ASSERT` actually used anywhere?

**Comment by abays** [comment]
Failed on PV creation.  Retrying `install-yamls-deploy`.

/test install-yamls-deploy

**Comment by abays** [comment]
recheck

File: Makefile (hunk 2 above)
Comment (kajinamit) [reply Makefile]: No it is no longer needed. I've removed it.

**Comment by abays** [comment]
recheck

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**